src/runtime/internal/dom.ts

```diff
@@ -1,4 +1,31 @@
 import { has_prop } from './utils';
+import { Fragment } from './types.js';
+
+// marks a part in the code where types
+// 1. should be improved or
+// 2. where casting is needed in order to satisfy TypeScript
+// 	a deeper look at these parts is needed to check if they can be replaced with a normal cast or if they currently contain a potential bug
+type TODO<T = any> = T
```

Comment:
Most things were typed after enabling `TypeScript`'s `strict` mode.
Those `TODO` casts are not always needed currently, but I think enabling strict mode for the future makes sense.